sentry/src/main/java/io/sentry/ITransactionListener.java

```diff
@@ -0,0 +1,11 @@
+package io.sentry;
+
+import org.jetbrains.annotations.ApiStatus;
+
+/** Used for performing operations when a transaction is started or ended. */
+@ApiStatus.Internal
+public interface ITransactionListener {
+  void onTransactionStart(ITransaction transaction);
+
+  void onTransactionEnd(ITransaction transaction);
```

Comment:
Since we have the method `transaction.finish()` I'd rather call it `onTransactionFinish`